drivers/console/Kconfig

```diff
@@ -210,6 +210,33 @@ config UART_PIPE_ON_DEV_NAME
 	  This option specifies the name of UART device to be used
 	  for pipe UART.
 
+config UPIPE_DEV_NAME_0
+	string "Device Name of UART Device 0 for pipe UART"
```

Comment:
Can genericize this description to "I/O device" instead of "UART device", since it should be possible to pass it a SPI interface in the future.